superset-frontend/src/SqlLab/components/ExploreResultsButton/index.jsx

```diff
@@ -149,28 +139,29 @@ class ExploreResultsButton extends React.PureComponent {
     );
   }
 
-  render() {
-    const allowsSubquery =
-      this.props.database && this.props.database.allows_subquery;
-    return (
-      <>
-        <Button
-          buttonSize="small"
-          onClick={this.props.onClick}
-          disabled={!allowsSubquery}
-          tooltip={t('Explore the result set in the data exploration view')}
-        >
-          <InfoTooltipWithTrigger
-            icon="line-chart"
-            placement="top"
-            label="explore"
-          />{' '}
-          {t('Explore')}
-        </Button>
-      </>
-    );
-  }
+  const allowsSubquery = props.database && props.database.allows_subquery;
```

Comment:
```suggestion
  const allowsSubquery = props.database?.allows_subquery;
```
Nit: This code can be cleaned with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).